pkg/cmd/render/render.go

```diff
@@ -251,7 +253,7 @@ func (r *renderOpts) Run() error {
 }
 
 func getDefaultConfigWithAuditPolicy() ([]byte, error) {
-	defaultPolicy, err := audit.DefaultPolicy()
+	defaultPolicy, err := libgoassets.DefaultPolicy("kube-apiserver-audit-policies", "openshift-kube-apiserver")
```

Comment:
why do we care about namespace and configmap name?

The default policy (still) depends on whether we have deleted old non-sha256 oauthaccesstokens (safe vs. non-safe policy). I think we ignored this here because kube-apiserver does not log oauthaccesstokens. But for a general `DefaultPolicy` functions we need it.

Reply:
Should:
- the ns and name parameter be removed and
- a conditional to pick secure or unsecure be added?

Reply:
I have renamed the DefaultPolicy function so that folks can tell the difference:
https://github.com/openshift/library-go/pull/1025/files#diff-e32a8349971df956b0a9ccb46a34e8a54f965b5d04e9a16a8a5d2d0d7dc005b6R26